package.json

```diff
@@ -6,165 +6,165 @@
-    "errorhandler": "^1.4.2",
-    "express": "~4.14.0",
-    "express-state": "^1.2.0",
-    "express-useragent": "^0.2.0",
+    "app-module-path": "1.0.2",
+    "autoprefixer-loader": "3.1.0",
+    "babel": "6.5.2",
+    "babel-cli": "6.11.4",
+    "babel-core": "6.13.2",
+    "babel-loader": "6.2.4",
+    "babel-plugin-add-module-exports": "0.1.4",
+    "babel-plugin-system-import-transformer": "2.0.0",
+    "babel-plugin-transform-decorators-legacy": "1.3.4",
+    "babel-plugin-transform-react-constant-elements": "6.9.1",
+    "babel-plugin-transform-react-display-name": "6.5.0",
+    "babel-plugin-transform-react-inline-elements": "6.8.0",
+    "babel-plugin-transform-runtime": "6.7.5",
+    "babel-plugin-typecheck": "3.8.0",
+    "babel-polyfill": "6.7.4",
+    "babel-preset-es2015": "6.13.2",
+    "babel-preset-react": "6.5.0",
+    "babel-preset-stage-0": "6.5.0",
+    "babel-register": "6.11.6",
+    "babel-runtime": "6.11.6",
+    "body-parser": "1.14.1",
+    "bootstrap-loader": "1.0.10",
+    "bootstrap-sass": "3.3.1",
+    "bootstrap-sass-loader": "1.0.9",
+    "bundle-loader": "0.5.0",
+    "cache-manager": "1.1.0",
+    "clean-webpack-plugin": "0.1.8",
+    "compression": "1.6.0",
+    "cookie-parser": "1.3.5",
+    "copy-to-clipboard": "1.0.4",
+    "cors": "2.7.1",
+    "crypto-js": "3.1.6",
+    "css-loader": "0.23.1",
+    "debug": "2.2.0",
+    "dotenv": "1.2.0",
+    "errorhandler": "1.4.2",
+    "express": "4.14.0",
+    "express-state": "1.2.0",
+    "express-useragent": "0.2.0",
     "extract-text-webpack-plugin": "2.0.0-beta.3",
-    "file-loader": "^0.8.4",
-    "fontfaceobserver": "^1.7.1",
-    "html-webpack-plugin": "^1.4.0",
-    "http-proxy": "^1.13.2",
-    "humps": "^1.0.0",
-    "imports-loader": "^0.6.3",
-    "jquery": "^2.1.4",
-    "json-loader": "~0.5.1",
-    "morgan": "^1.6.1",
-    "node-sass": "^3.2.0",
-    "normalizr": "^2.0.0",
-    "phantomjs": "^1.9.18",
-    "pretty-error": "^2.0.0",
-    "promise": "^7.0.4",
-    "proxy-middleware": "^0.14.0",
-    "qs": "^6.1.0",
-    "raven": "^0.11.0",
-    "raw-loader": "^0.5.1",
-    "react": "^0.14.8",
-    "react-bootstrap": "^0.29.5",
-    "react-cookie": "^0.3.4",
-    "react-dom": "^0.14.0",
-    "react-helmet": "^3.1.0",
-    "react-metrics": "^1.1.0",
-    "react-paginate": "^0.4.0",
-    "react-redux": "^4.4.1",
-    "react-router": "^2.4.1",
-    "react-router-bootstrap": "^0.20.1",
-    "react-router-redux": "^4.0.0",
-    "react-router-scroll": "^0.2.0",
-    "react-scroll": "^1.0.4",
-    "redux": "^3.3.1",
-    "redux-connect": "^2.4.0",
-    "reselect": "^2.5.1",
-    "resolve-url": "^0.2.1",
+    "file-loader": "0.8.4",
+    "fontfaceobserver": "1.7.1",
+    "html-webpack-plugin": "1.4.0",
+    "http-proxy": "1.13.2",
+    "humps": "1.0.0",
+    "imports-loader": "0.6.3",
+    "jquery": "2.1.4",
+    "json-loader": "0.5.1",
+    "morgan": "1.6.1",
+    "node-sass": "3.8.0",
+    "normalizr": "2.0.0",
+    "phantomjs": "1.9.18",
```

Comment:
I think you commented on an old commit of `package.json` because it's already `1.9.20` in this branch.